src/sbt-test/semver/filters/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Created by .ignore support plugin (hsz.mobi)
+### Scala template
+*.class
+*.log
+
+# sbt specific
+.cache
+.history
+.lib/
+dist/*
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+global/
+pending*
+
+# Scala-IDE specific
+.scala_dependencies
+.worksheet
```

Comment:
I usually suggest these configs go into your `~/.gitignore_global` file

Reply:
Completely agree, I think I blindly copied it from some other test. I can update the PR on Monday.

Reply:
I removed IDE-specific part from `.gitignore`